doc/source/raysgd/v2/architecture.rst

```diff
@@ -0,0 +1,44 @@
+:orphan:
+
+.. _sgd-arch:
+
+Architecture
+============
+
+A diagram of the RaySGD architecture is provided below.
+
```

Comment:
@matthewdeng do you have a diagram that I can add here (from lucidcharts, preferably a svg?)